<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDialog</class>
 <widget class="QMainWindow" name="AccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>416</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>424</width>
    <height>165</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>611</width>
      <height>71</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(225, 255, 251);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="SetButton">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>11</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>手动设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReturnButton">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>11</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>387</x>
       <y>11</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
     <property name="shortcut">
      <string>Enter</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>11</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
     <property name="shortcut">
      <string>Esc</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labeluserSet">
         <property name="text">
          <string>    用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtuser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelpasswordSet">
         <property name="text">
          <string>      密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtpassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="poplabel">
         <property name="text">
          <string> POP服务器：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtpopserver"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSSLpop">
         <property name="text">
          <string>SSL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="popportlabel">
         <property name="text">
          <string>端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="popportEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="smtplabel">
         <property name="text">
          <string>SMTP服务器：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtsmtpserver"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSSLsmtp">
         <property name="text">
          <string>SSL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="smtpportlabel">
         <property name="text">
          <string>端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="smtpportEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SetButton</sender>
   <signal>clicked()</signal>
   <receiver>AccountDialog</receiver>
   <slot>showManualSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>AccountDialog</receiver>
   <slot>onLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AccountDialog</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ReturnButton</sender>
   <signal>clicked()</signal>
   <receiver>AccountDialog</receiver>
   <slot>hideManualSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLogin()</slot>
  <slot>onCancel()</slot>
  <slot>hideManualSet()</slot>
  <slot>showManualSet()</slot>
  <slot>onSetpopSLL()</slot>
  <slot>onSetsmtpSSL()</slot>
 </slots>
</ui>
